templates/zerver/api/get-messages.md

```diff
@@ -26,7 +25,9 @@ whether there are more messages matching the query that were not
 returned due to the `num_before` and `num_after` limits.
 
 We recommend using `num_before <= 1000` and `num_after <= 1000` to
-avoid generating very large HTTP responses.
+avoid generating very large HTTP responses. A maximum of 5000 messages
+can be obtained per request, attempting to exceed this will result in an
```

Comment:
Can you change the comma here to a semicolon?  Or just make a new sentence starting with "Attempting to exceed...".